roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -175,6 +175,13 @@
   notify: Restart crio
   when: http_proxy is defined or https_proxy is defined
 
+- name: Cri-o | write cri-o options drop-in
+  template:
+    src: options.conf.j2
+    dest: /etc/systemd/system/crio.service.d/options.conf
+    mode: 0644
+  notify: Restart crio
+
```

Comment:
Why not use `file` instead of template and condition the whole task ?
No need for the drop-in file at all if you don't enable checkpointing, right ?

Reply:
> No need for the drop-in file at all if you don't enable checkpointing, right ?

Right. The reason is to reserve the possibility for adding other options of crio in the future, and actually restarting crio here more once haven't any side-effects.

Reply:
> The reason is to reserve the possibility for adding other options of crio in the future,

No need for it right now then correct ?

Reply:
good point. Done